### ISSUE TITLE
About Character Perceptual Loss Function

### ISSUE THREAD

**Issue by INWLY** [yeungchenwa/HDR#5]
Hello Zhenhua Yang,
I am very interested in the **_Character Perceptual Loss Function_** proposed in the article. I noticed that the training code for the model is not provided in this repository. I would like to ask if you can provide the design details of CPLoss?

I sincerely look forward to your reply~
